packages/react/src/components/Grid/Column.js

```diff
@@ -36,6 +36,7 @@ function Column({
 }
 
 const spanPropType = PropTypes.oneOfType([
+  PropTypes.bool,
   PropTypes.number,
   PropTypes.shape({
     span: PropTypes.number,
```

Comment:
The object format should probably support booleans too
```suggestion
     span: PropTypes.oneOfType([PropTypes.bool, PropTypes.number]),
```